### PR TITLE
alternator: Add build byproducts to .gitignore

### PR DESCRIPTION
**Pull request by dekimir**
Add .pytest_cache and expressions.tokens to the top-level .gitignore.

@psarna @nyh

### CODE REVIEW

**Comment by tgrabiec** [comment]
@psarna What is the question?

**Comment by psarna** [comment]
... github interface is just perfect. Let me write it again and click a proper button this time

File: .gitignore [hunk 1]
```diff
@@ -19,3 +19,5 @@ CMakeLists.txt.user
 __pycache__CMakeLists.txt.user
 .gdbinit
 resources
+.pytest_cache
+/expressions.tokens
```
Comment (psarna) [inline .gitignore]: why the preceding `/`?

Comment (denesb) [reply .gitignore]: Presumably to only ignore that file if its in the top-level directory.

Comment (psarna) [reply .gitignore]: Ok. This question was non-blocking anyway, I approve the changes.

Comment (dekimir) [reply .gitignore]: Yes, I want to defer to the future any decision to ignore this filename everywhere. It's preferable to only ignore it in root for now.

**Pull request merged**